src/components/LabeledSwitch/LabeledSwitch.jsx

```diff
@@ -0,0 +1,126 @@
+import _ from 'lodash';
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createLucidComponentDefinition }  from '../../util/component-definition';
+import Switch from '../Switch/Switch';
+
+const boundClassNames = lucidClassNames.bind('&-LabeledSwitch');
+const {
+	bool,
+	func,
+	node,
+	object,
+	string
+} = React.PropTypes;
+
+/**
+ * {"categories": ["controls", "toggles"], "extend": "Switch"}
```

Comment:
Add `"madeFrom": ["Switch"]` as well.

You should spread the prop types from `Switch` if you want to extend it.